File: app/src/main/java/tr/name/fatihdogan/books/api/response/Volume.java
package tr.name.fatihdogan.books.api.response;

import android.support.annotation.Keep;

@Keep
@SuppressWarnings({"WeakerAccess", "unused"})
public class Volume {

    public static final String KIND_BOOKS_VOLUME = "books#volume";

    /**
     * {@link #KIND_BOOKS_VOLUME}
     */
    public String kind;
    public String id;
    public String etag;
    public String selfLink;
    public VolumeInfo volumeInfo;
    public UserInfo userInfo;
    public SaleInfo saleInfo;
    public AccessInfo accessInfo;
    public SearchInfo searchInfo;
}

File: app/src/main/java/com/fthdgn/books/api/response/Volume.kt
@file:Suppress("unused")

package com.fthdgn.books.api.response

import android.support.annotation.Keep

@Keep
class Volume {

    /**
     * [.KIND_BOOKS_VOLUME]
     */
    var kind: String? = null
    lateinit var id: String
    var etag: String? = null
    var selfLink: String? = null
    var volumeInfo: VolumeInfo? = null
    var userInfo: UserInfo? = null
    var saleInfo: SaleInfo? = null
    var accessInfo: AccessInfo? = null
    var searchInfo: SearchInfo? = null

    companion object {

        val KIND_BOOKS_VOLUME = "books#volume"
    }
}
